docs/source/how_to_guides/optimization/how_to_handle_errors_during_optimization.ipynb

```diff
@@ -0,0 +1,286 @@
+{
```

Comment:
<p>occur not occurr</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/OpenSourceEconomics/estimagic/pull/339/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='0'/>

Reply:
Done.